unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenTipsController.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+namespace DCL.LoadingScreen
+{
+    /// <summary>
+    /// Loading screen tips provider. The responsibility of this class is to create the loading tips and provide them as needed
+    /// </summary>
+    public class LoadingScreenTipsController
+    {
+        private List<LoadingTip> currentLoadingTips;
+        private int currentRandomIndex;
+
+        public void LoadDefaults()
+        {
+            var mana = new LoadingTip("MANA is Decentraland’s virtual currency. Use it to buy LAND and other premium items, vote on key policies and pay platform fees.", "TipsImages/ManaImg");
```

Comment:
When I look at I have a craving for serialising it and having "strong" references to srpites

Reply:
Serialized in the view!